src/controllers/confirm.company.controller.ts

```diff
@@ -42,9 +42,12 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
 
       let latestGaz1Date: string;
       const companyEligibility: CompanyEligibility = await getCompanyEligibility(company.companyNumber, token);
-      if (companyEligibility.eligibility_status === EligibilityStatus.ELIGIBLE ||
-        companyEligibility.eligibility_status === EligibilityStatus.INELIGIBLE_GAZ2_REQUESTED) {
+
+      if (companyEligibility.eligibility_status === EligibilityStatus.ELIGIBLE) {
```

Comment:
This is a personal preference one, but nodejs will allow you to capture just the fields you want from an object when it is being assigned.
instead of 
`const companyEligibility: CompanyEligibility = await getCompanyEligibility(company.companyNumber, token);`
you can have `const {eligibility_status} = await getCompanyEligibility(company.companyNumber, token);`
then you can just use eligibility_status in the if statements. It will reduce the code but it's just a different way i guess, down to what you prefer.

Reply:
Simplified the code.